README.md

```diff
@@ -148,73 +148,29 @@ _Note that options were changed in version 0.5.3_
 For example these commands:
 
 ```python
-worldengine world -s 4 -n an_example -p 2048 -q 25 -x 2048 -y 2048
+worldengine -b -s 4 -n an_example -q 25 -x 2048 -y 2048
```

Comment:
ok for removing `world` but perhaps we could avoid using `-b` here because it just changes the format we use for saving the world, so there is no visible effect here